Samples/Windowing/cs-winui/SampleApp (Package)/SampleApp (Package).wapproj

```diff
@@ -42,7 +42,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
     <EntryPointProjectUniqueName>..\SampleApp\SampleApp.csproj</EntryPointProjectUniqueName>
-    <PackageCertificateThumbprint>52902158A7BB4EF57D9DB04AF265DB38A11F1661</PackageCertificateThumbprint>
+    <PackageCertificateThumbprint>697B0A4425F6E61C280403E9B3F468E3E3C905A5</PackageCertificateThumbprint>
```

Comment:
Is this change intentional, Erik?

Reply:
Yes, intentional, although maybe optional? When building, VS was giving warnings about the certificate being expired. So I regenerated it. 
They were warnings though, not errors, so if you think I should undo this change, I will.

Reply:
I wonder if we should remove this and set `AppxPackageSigningEnabled` to False like in this sample: https://github.com/microsoft/WindowsAppSDK-Samples/blob/c8070a0d1710d19f6e870f2d3ec12e5422b3da90/Samples/ResourceManagement/cs-winui/winui_desktop_packaged_app%20(Package)/winui_desktop_packaged_app%20(Package).wapproj#L43

Although that is a separate issue.

Reply:
I actually do think that is a good idea. Having it be true, means that in the pipeline build it will fail, so when running the pipeline I have to do something like:
```MsbuildArgs: '/p:VCToolsInstallDir="$(VCToolsInstallDir)\" /p:AppxPackageSigningEnabled=false'```
If we can avoid it in the samples themselves, that would reduce a little the churn we have... 
what do you think?

Reply:
I agree that we should turn off signing and follow the example from the existing ResourceManagement samples